src/App.tsx

```diff
@@ -1,26 +1,177 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState, useEffect } from 'react';
+import { addTodo, deleteTodo, getTodos, updateTodo } from './api/todos';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Errors } from './components/Errors';
+
+import { loadingObject } from './utils/loadingObject';
+import { filteredTodos } from './utils/filteredTodos';
+
+import { Todo } from './types/Todo';
+import { ErrorMessage } from './types/Errors';
+import { Loading } from './types/Loading';
+import { Filters } from './types/Filters';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | string>('');
```

Comment:
it will be safer that way
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorMessage>(ErrorMessage.Default);
```